kDriveCore/Data/Cache/AccountManager.swift

```diff
@@ -74,7 +73,7 @@ public protocol AccountManageable: AnyObject {
     func loadAccounts() -> [Account]
     func saveAccounts()
     func switchAccount(newAccount: Account)
-    func setCurrentDriveForCurrentAccount(drive: Drive)
```

Comment:
Passing a random drive was causing issues, now using the drive ID (like done internally before)